cnv/cnv_users_guide/cnv-create-vms.adoc

```diff
@@ -0,0 +1,33 @@
+[id="cnv-create-vms"]
+= Creating virtual machines
+include::modules/cnv_document_attributes.adoc[]
+:context: cnv-create-vms
+toc::[]
```

Comment:
Your TOC isn't rendering for me.

Reply:
Let's follow-up on this one after the topic map file is updated early this week. I would like to get the outstanding PRs merged early Monday. There will definitely be more changes following QE review but I'd like to get the largest amount of changes merged sooner than later.